src/renderers/webgl/WebGLShadowMap.js

```diff
@@ -256,6 +258,8 @@ function WebGLShadowMap( _renderer, _objects, maxTextureSize ) {
 
 		scope.needsUpdate = false;
 
+		_renderer.setRenderTarget( currentRenderTarget );
```

Comment:
I'm a bit confused about your scoping since you refer to variable that is create inside an if statement. What if there are no lights in the scene? You would set `_renderer.setRenderTarget( undefined ); `, right?

Reply:
Oh! Overlooked that, sorry

Reply:
Fixed